v4/test/auth/auth_test.go

```diff
@@ -0,0 +1,257 @@
+package test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/weaviate/weaviate-go-client/v4/test/testsuit"
+
```

Comment:
rm imports newline. maybe we can add a linter to the ci pipeline

Reply:
gofumpt will only properly sort if there are no line breaks in the import block 🙃 but good to know you are tracking a linter issue.

Reply:
ok, now